### PR TITLE
Added AdminPermissionsForUserScreensTest

### PR DESCRIPTION
**Pull request by SVar25**
https://trello.com/c/s1OW9njz/880-admin-permission-settings-for-users-screens-tc

https://trello.com/c/rHctRFDO/841-admin-permission-settings-for-users-screens

### CODE REVIEW

File: src/test/java/model/admin/PermissionConfigurePage.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package model.admin;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class PermissionConfigurePage extends ConfigurationPage {
+
+    @FindBy(xpath = "//tbody/tr[1]/td[3]/input[1]")
+    private WebElement checkPermissionButton;
```
Comment (oooleks) [inline src/test/java/model/admin/PermissionConfigurePage.java:10]: It's not clear which exactly check-box it belongs to: we've got 4 Permission pages with checkboxes:
- Entities;
- Screens;
- Menus;
- UI

File: src/test/java/model/admin/ConfigurationPage.java [hunk 2]
```diff
@@ -169,4 +180,21 @@ public ConfigurationPage waitForCmdHistory(String text) {
         getWait().until(ExpectedConditions.textToBePresentInElement(cmdHistory, text));
         return this;
     }
+
+    public ConfigurationPage clickNewScreens() {
```
Comment (oooleks) [inline src/test/java/model/admin/ConfigurationPage.java:184]: Does it really return the same ConfigurationPage on clickNewScreens()?

File: src/test/java/test/admin/AdminPermissionsForUserScreensTest.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                .goToLoginPage(instanceCreds[0]);
+        loginPage.login(adminLoginCred, instanceCreds[1]);
+
+        new ConfigurationPage(getDriver())
+                .clickConfiguration()
+                .clickNewPersona();
+
+        PersonasConfigurePage createPersona = new CreatePersonasPage(getDriver())
+                .createPersona(PERSONA_NAME);
+        Assert.assertTrue(createPersona.isPersona(PERSONA_NAME));
```
[the remark this answers is not in the document]
Comment (SVar25) [reply src/test/java/test/admin/AdminPermissionsForUserScreensTest.java:37]: This assert doesn't work

[the remark this answers is not in the document]
Comment (oooleks) [reply src/test/java/test/admin/AdminPermissionsForUserScreensTest.java:37]: It working - the only change I did not create the variable, so check this for errors, typo probably:
Assert.assertTrue(new CreatePersonasPage(getDriver())
                .createPersona(PERSONA_NAME).isPersona(PERSONA_NAME));

File: src/test/java/model/admin/PermissionConfigurePage.java [hunk 4]
```diff
@@ -35,9 +35,8 @@ public boolean checkBoxIsSelected() {
         if (checkBox.isSelected()) {
             return true;
         } else {
-
+            return false;
         }
```
Comment (oooleks) [inline src/test/java/model/admin/PermissionConfigurePage.java]: No `else` with return needed and you can return `false` at the end of the method.

File: src/test/java/test/admin/AdminPermissionsForUserScreensTest.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (SVar25) [reply src/test/java/test/admin/AdminPermissionsForUserScreensTest.java:32]: Thank you

**Review by oooleks — APPROVED** [review]

**Review by MaxBurdin — APPROVED** [review]

**Pull request merged**